.github/workflows/nightly-CI.yml

```diff
@@ -16,8 +16,8 @@ jobs:
       # is too noisy without a proper allow failure option.
       matrix:
         version:
-          - 1.3
-          - 1
+          - "1.3"  # Invenia Prod Version
```

Comment:
do we even need this entire file at all? Seems to just be duplicating some of the `CI.yml` stuff as both run at night, etc.

Reply:
Not sure about your question, but we should switch "1.3" to "1.5"

Reply:
Ah yeah, a lot of these similar MR's are going to have `1.3` as they were made before we decided to go to `1.5`.

Reply:
Still needs updating to 1.5

Reply:
only if we assume we should be keeping this file.

Reply:
Just deleted this file. Don't think it needs to exist and nobody has told me otherwise.